hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieInternalRowUtils.scala

```diff
@@ -43,25 +43,9 @@ object HoodieInternalRowUtils {
     ThreadLocal.withInitial(new Supplier[HashMap[(StructType, StructType), UnsafeProjection]] {
```

Comment:
Why using ThreadLocal for this Map, while ConcurrentHashMap for others?

Reply:
UnsafeProjection is not thread safe

Reply:
Comment is already been leaved.